content/03.methods.md

```diff
@@ -35,22 +35,43 @@ These articles were the first 200 English language "Journal" articles returned b
-Since many last or first authorships may be non-names, we additionally filtered out any identified names if they partially or fully match any of the following terms: "consortium", "group", "initiative", "team", "collab", "committee", "center", "program", "author", or "institute".
-Furthermore, since many articles only contain first and last name initials, we remove any names less than four letters with a  "." or "-".
-Finally, we only consider any remaining names of more than two characters.
-We do this additional filtering to remove any web-scraping or coreNLP errors that only return partial names.
- 
-Since gender and origin predictions require the speaker’s full name, when possible, a longer name with minimal edit (Levenshtein) distance mentioned within the same article replaces the coreNLP predicted speaker.
+### Name Formatting
+
+#### Name Formatting for Gender Prediction in Quotes or Mentions
+To identify the gender of a quoted or mentioned person, we first attempt to identify the person’s full name.
+Even though genderizeR only uses the first name to make the gender prediction, identifying the full name gives us greater confidence that we are using the first name.
+To identify the full name, we take the predicted speaker by coreNLP and match it to the longest matching name within the same article.
+We match names by finding the longest mentioned name in the article with minimal edit (Levenshtein) distance.
 The name with the smallest edit distance, where character deletions have zero cost, is defined as the matching name.
 Character deletion was assigned a zero cost because we would like exact substring matches.
 For example, the calculated cost, including a cost for character deletion, between John and John Steinberg is 10; without character deletion, it is 0.
 Compared with the distance between John and Jane Doe, with character deletion cost, it is 7; without it is 2.
-Furthermore, the identification of simultaneously cited and quoted speakers used this same name matching strategy.
- 
+If we are still unable to find a full name, or if coreNLP cannot identify a speaker at all, we also look if coreNLP identified a gendered pronoun to the quote.
+If so, we assign the gender of the speaker to the gender of the pronoun.
+We ignore all quotes with no name or partial names and no associated pronouns.
+
+
+#### Name Formatting for Gender Prediction of Authors
+Only articles with more than one author were considered.
+Similarly for the name formatting for quotes, we need to extract the first name of the authors.
+Names were cast to lowercase and processed using the R package humaniformat [@https://cran.r-project.org/web/packages/humaniformat/index.html].
+humaniformat identifies if names are reversed (Lastname, Firstname), as well as identifies middle names.
+This processing was not done in the quotes prediction because we assume the names written in an article are not reversed or abbreviated.
```

Comment:
```suggestion
This processing was not required for quote prediction because we names written in news articles did not appear to be reversed or abbreviated.
```